manifests/poc-data-cluster/prod/spark/hotspot-rewards-gold.yaml

```diff
@@ -0,0 +1,76 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: hotspot-rewards-gold-query
+  namespace: spark
+data:
+  query.sql: |
+    SELECT 
+      max(end_period) as most_recent_ts, 
```

Comment:
Please use 'date' and not end_period.

Reply:
There's no date output from the silver table